integration/src/integration/java/pl/allegro/tech/hermes/integration/IntegrationTest.java

```diff
@@ -35,4 +36,9 @@ public void initializeIntegrationTest() {
                 CONFIG_FACTORY);
     }
 
+    @AfterMethod
+    public void after() {
+        management.query().querySubscriptions("{\"query\": {}}").forEach(sub ->
```

Comment:
We decided to remove subscription after each test because right now at the end of suit execution, single test execution handles more then 70 subscriptions , which is very inefficient, especially on travis.